settings.py

```diff
@@ -0,0 +1,5 @@
+# game setup
```

Comment:
Good to start, but we will have complications if we ever want to load these settings from a config file. We can continue to use this for now until that happens.